### ISSUE TITLE
Create a new tag?

### ISSUE THREAD

**Issue by sunshineo** [gali8/Tesseract-OCR-iOS#249]
4.0.0 does not work with Xcode7
It's fixed here already.
I've been installing from this github repo directly for a while.
Can we create a new tag?

**Comment by mweibel** [comment]
:+1: would be awesome

**Comment by ws233** [comment]
@sunshineo, do you mean new CocoaPod release or what?

**Comment by sunshineo** [comment]
Yes. But they start with create a new tag here in Github, right?
